tests/translate/csharpgen.py

```diff
@@ -0,0 +1,89 @@
+import pythonir
+
+
+# a small helper function for generating C# code
+class CSharpGenerator:
+    def __init__(self, out_file):
+        self.out_file = out_file
+        self.tab_depth = 0
+
+    def get_line_prefix(self):
+        return self.tab_depth * 4 * " "
+
+    def writeln(self, line):
+        self.out_file.write(f"{self.get_line_prefix()}{line}\n")
+
+    def open_block(self):
+        self.writeln("{")
+        self.tab_depth += 1
+
+    def close_block(self):
+        self.tab_depth -= 1
+        self.writeln("}")
+
+
+def translate(module, filename):
+    with open(filename, "w") as file:
+        gen = CSharpGenerator(file)
+        translate_to(module, gen)
+
+
+def translate_to(module, gen):
+    # first get all imports
+    # then generate project/assembly references and/or comments in the
+    # generated code
+    # for now just hard-code them
+    # Note that these will change depending on the .NET version we are targeting
+    references = [
```

Comment:
references?

Reply:
I've renamed it to `assembly_references`.  These will need to be infered from the imports but for now I'm just hard-coding them.